docs/wip/osm_configmap.md

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
Could you move this file to docs/content/docs/osm_config_map.md so we can easily see what the diffs are?